src/components/OptimizedImage.js

```diff
@@ -55,8 +56,8 @@ export default function OptimizedImage(props) {
           inset: 0,
           objectFit,
           width: 'calc(100%)', // width for image rotation
-          height: 'calc(100%)', // height for image rotation
-          transform: `rotate(${rotation}deg)`,
+          height: 'calc(99%)', // height for image rotation
```

Comment:
We can remove the `calc` from the height and width if they're just percentages now. We used to adjust the size by a couple of pixels to remove random gaps, but that's gone now. The comments don't make much sense either now.